src/jest/environment.ts

```diff
@@ -0,0 +1,46 @@
+import NodeEnvironment from 'jest-environment-node';
+
+import { AugmentedDriver, Config, buildDriver } from '../selenium';
+
+class SeleniumEnvironment extends NodeEnvironment {
+  config: Config;
+  global: {
+    browser: AugmentedDriver<unknown>;
+  };
+
+  constructor(config: { testEnvironmentOptions: Config }) {
+    super(config);
+    this.global = this['global'] || {};
+    this.config = config.testEnvironmentOptions;
+  }
+
+  async setup() {
+    await super.setup();
+
+    this.global.browser = await buildDriver(this.config);
+  }
+
+  async teardown() {
+    const { browser } = this.global;
+    if (browser) {
+      // https://github.com/alexeyraspopov/jest-webdriver/issues/8
+      try {
+        await browser.close();
+      } catch (e) {
+        /* ignore */
+      }
+
+      // https://github.com/mozilla/geckodriver/issues/1151
+      try {
+        await browser.quit();
+      } catch (e) {
+        /* ignore */
+      }
+    }
+
+    await super.teardown();
+  }
+}
+
+// export in a way that is ingestible by jest
+module.exports = SeleniumEnvironment;
```

Comment:
I also am not sure why I need to do this. if I do `export default class` it gets translated to `exports.default =`

Reply:
even with fix to letting babel compile? `add-module-exports` should handle this

Reply:
I don't get why isn't that the default behavior? how do all the babel-compiled libs out there actually work...

Reply:
so what do you suggest? module.exports or the plugin? I'd be inclined on the latter. should even be upstream imo. otherwise all node packages would be broken. wdyt?

Reply:
just to be clear `add-module-exports` _is_ upstream in our babel preset

Reply:
[is it?](https://github.com/4Catalyzer/javascript/search?q=add-module-exports&unscoped_q=add-module-exports)

Reply:
wait...WHAT...that's so weird...it must just be in my personal version https://github.com/jquense/babel-preset-jason/blob/master/index.js#L98 oops I guess b/c i wanted to encourage no babel for node